snap/snapcraft.yaml

```diff
@@ -25,6 +25,7 @@ parts:
   solidity:
     source: .
     source-type: git
+    source-branch: release
```

Comment:
I don't think this is a good solution. I would leave this as is, using the default develop branch. This will give us a constantly updatad snap on the edge channel with the things that land in develop.

Then, we can use launchpad to publish a snap from the release branch to beta or candidate, for a little testing before manually moving it to stable.

Reply:
So, do we need to change the grade option to "edge" instead of stable?

Reply:
not necessarily. Grade in there is a guardrail. If it's not stable, it won't allow the snap to be moved to stable. build.snapcraft.io will always publish the snap to edge.